.github/workflows/ci.yml

```diff
@@ -64,7 +63,7 @@ jobs:
       # "master" branch, we allow installing pre-releases with the pip command.
       - name: Allow pre-releases for master branch
         if: github.ref == 'refs/heads/master'
-        run: echo "TOX_COMMAND=${{ env.TOX_COMMAND }} --pre" >> $GITHUB_ENV
+        run: echo "tox --pre" >> $GITHUB_ENV
```

Comment:
I am not sure this will be working since you removed variable assignment. You are not setting any environment variable with this and after the change the echo would not do anything. The consequence of this not working is that QA deploys would not include prerelease packages.

Reply:
Nice catch. In tox 4+ it is recommended to set PIP_PRE environmental variable as --pre is deprecated and could even not work.